Libraries/LibWebView/Settings.cpp

```diff
@@ -202,6 +230,30 @@ JsonValue Settings::serialize_json() const
 
     settings.set(do_not_track_key, m_do_not_track == DoNotTrack::Yes);
 
+    // dnsSettings :: { mode: "system" } | { mode: "custom", server: string, port: u16, type: "udp" | "tls", forciblyEnabled: bool }
+    JsonObject dns_settings;
+    m_dns_settings.visit(
+        [&](SystemDNS) {
+            dns_settings.set("mode"sv, "system"sv);
+        },
+        [&](DNSOverTLS const& dot) {
+            dns_settings.set("mode"sv, "custom"sv);
+            dns_settings.set("server"sv, dot.server_address.view());
+            dns_settings.set("port"sv, dot.port);
+            dns_settings.set("type"sv, "tls"sv);
+            dns_settings.set("local_dnssec"sv, false);
```

Comment:
`localDNSSEC` or something else non-snakey pls

Reply:
oh that one isn't supposed to be in there (yet), will delete.